webapp/src/views/SamplePrime.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <Navbar />
+
+  <div id="tableContainer" class="container">
+    <div class="row">
+      <div class="col-sm-12 mx-auto">
+        <SampleTablePrime />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Navbar from "@/components/Navbar";
+import SampleTablePrime from "@/components/SampleTablePrime";
+
+export default {
+  name: "Samples",
+  components: {
+    Navbar,
+    SampleTablePrime,
+  },
+};
+</script>
+
+<style scoped>
+.fade {
+  opacity: 0;
+  transition: opacity 0.15s linear;
+}
+.fade.show {
+  opacity: 1;
+}
```

Comment:
I don't think this is doing anything (these styles are use in Modal.vue, but I think they are copied to several of our pages, Equipment.vue, Samples.vue, etc... . Probably my mistake a while ago. It would be nice if our linter could flag unused styles)